lib/tasks/migrate_assets.rake

```diff
@@ -0,0 +1,134 @@
+# frozen_string_literal: true
+
+require "logger"
+
+namespace :migrate_assets do
+  desc "Run all tasks"
+  task to_active_storage: :environment do
```

Comment:
@tachyons we are using [rails-data-migrations](https://www.rubydoc.info/gems/rails-data-migrations/1.0.4) gem instead of [data-migrate](https://github.com/ilyakatz/data-migrate), so there isn't any option to redo the migrations.

![Screenshot 2023-06-01 at 7 11 11 PM](https://github.com/CircuitVerse/CircuitVerse/assets/85568177/b3c31520-14e1-4a84-b838-968de1adf8d8)
Even with `rake data:migrate:up VERSION=3424...` it gives error